edc-extensions/edr/edr-cache-sql/src/test/java/org/eclipse/tractusx/edc/edr/store/sql/PostgresqlTransactionalStoreSetupExtension.java

```diff
@@ -46,14 +49,17 @@
 public class PostgresqlTransactionalStoreSetupExtension implements BeforeEachCallback, AfterEachCallback, BeforeAllCallback, ParameterResolver {
 
     private final String datasourceName;
+    private final QueryExecutor queryExecutor;
+    private final Monitor monitor = mock(Monitor.class);
```

Comment:
just a nit: it's not necessary to pass the class to the `mock()` method if the type is explicitly declared